### ISSUE TITLE
import routers using ES2015 modules

### ISSUE THREAD

**Issue by nbrady-techempower** [juliancwirko/meteor-s-alert#54]
In a current project I'm blowing up globals in a lib file to test imports which I noted [here](https://forums.meteor.com/t/destroying-globals-to-ensure-module-imports-work/21336). This has been working great so far and is a good way to prepare for the upcoming move to strictly using imports. Unfortunately, sAlert breaks when I blow up the FlowRouter global because it is not being imported in s-alert.js. This would be super helpful and prepare for the move to ES2015 module imports!

**Comment by juliancwirko** [comment]
Yes, I plan to rebuild the package. But not now. Also I don't know exactly how to replace this part: https://github.com/juliancwirko/meteor-s-alert/blob/master/client/s-alert.js#L132 This is based on globals, and I don't want to import routers here. I just want to check if someone use them. Maybe I will remove this functionality and provide some docs how to clear alerts on route change.
 
For now in Meteor 1.3 the package should work. Of course if you don't block (intencionally) 'Meteor' and other globals ;)

**Comment by juliancwirko** [comment]
Oh, and if you use React you can try pure React component: https://www.npmjs.com/package/react-s-alert

**Comment by nbrady-techempower** [comment]
Oh very cool. I am thinking about moving to React soon. Thanks for the updates!
